packages/config/readme.md

```diff
@@ -1,4 +1,126 @@
-
 # Hops Config
 
-T.B.D. (To Be Documented)
+hops-config exposes options to configure the other Hops packages and Webpack.
+
+# Installation
+When you install [hops-cli](https://www.npmjs.com/package/hops-cli), hops-config will be installed as a dependency as well - so normally, you don't have to separately install hops-config (and we don't recommend that). If you for some reason need to do that anyways, just do:
+
+```
+npm install --save hops-config
+```
+
+# Usage
+
+Please find a list of all available options below. They can be set in your project's package.json and can be overridden by using [npm config](https://docs.npmjs.com/cli/config) in the command line. It's important to understand that the configuration is immutable at runtime, i.e. you can't for example import values and change them. hops-config provides an `extends` field as a means for augmenting the existing configuration.
```

Comment:
I think I'd get rid of the half sentence starting with "i.e." - I think they're a bit confusing, to be honest.

Additionally, we might want to address the fact that `hops-config` can not just be extended, but also imported in downstream projects: its isomorphic importability is something that I rather like. What do you think?